how-to-guides/cloud-services/aws/connect-to-a-documentdb-cluster.md

```diff
@@ -0,0 +1,96 @@
+![image](../../../.gitbook/assets/documentdb-architecture.png)
+
+## Existing Database
+
+### Step 1: Accessing DocumentDB from a DevBox
+
+1. Follow the [Connecting to AWS](../../existing-network/connecting-to-aws.md) guide.
+2. Go to **Amazon DocumentDB > Clusters > Your Cluster.**
+3. Select **Connectivity & Security.** You will see the commands and DocumentDB hostname for accessing it.
+
+![image](../../../.gitbook/assets/documentDB-connectivity-and-security.png)
+
+4. Go to **DevBox**.
+5. Import the MongoDB public GPG key:
+```sh
+curl -fsSL https://www.mongodb.org/static/pgp/server-7.0.asc | \
+   sudo gpg -o /usr/share/keyrings/mongodb-server-7.0.gpg \
+   --dearmor
+```
+6. Create a list file for MongoDB:
+```sh
+echo "deb [ arch=amd64,arm64 signed-by=/usr/share/keyrings/mongodb-server-7.0.gpg ] https://repo.mongodb.org/apt/ubuntu jammy/mongodb-org/7.0 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-7.0.list
+```
+7. Reload local package database:
+```sh
+sudo apt-get update
+```
+8. Install mongo shell:
+```sh
+sudo apt-get install -y mongodb-org
+```
+9. Download certificate:
+```sh
+wget https://truststore.pki.rds.amazonaws.com/global/global-bundle.pem
+```
+10. Connect to mongo:
+```sh
+mongosh --tls --host <your-cluster-endpoint>:27017 --tlsCAFile /path/to/global-bundle.pem --username <your-username> --password <your-password>
+```
+where the cluster endpoint will be available in **Connectivity & Security** section.
+
+![image](../../../.gitbook/assets/documentdb-access.png)
+
+
+## New Database
+
+### Step 1: Creating a DocumentDB Instance
+
+1. Follow the [Connecting to AWS](../../existing-network/connecting-to-aws.md) guide.
+2. Go to **Amazon DocumentDB > Create Cluster**.
+3. Select **Instance Based Cluster**.
+4. In the **Configuration** section choose **Instance Class**, number of instances for your cluster.
+5. Set **Username and Password**
+6. Turn on **Show advanced settings**.
+7. In the **Network settings** , choose the VPC where your bastion host is running, subnet group and assign a security group which has inbound access to only VPC.
+8. Click **Create cluster**.
+
+![image](../../../.gitbook/assets/documentdb-cluster.png)
+
+### Step 2: Accessing DocumentDB from a DevBox
+
+1. Go to **DevBox**.
+2. Import the MongoDB public GPG key:
+```sh
+curl -fsSL https://www.mongodb.org/static/pgp/server-7.0.asc | \
+   sudo gpg -o /usr/share/keyrings/mongodb-server-7.0.gpg \
+   --dearmor
+```
+3. Create a list file for MongoDB:
+```sh
+echo "deb [ arch=amd64,arm64 signed-by=/usr/share/keyrings/mongodb-server-7.0.gpg ] https://repo.mongodb.org/apt/ubuntu jammy/mongodb-org/7.0 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-7.0.list
+```
+4. Reload local package database:
+```sh
+sudo apt-get update
+```
+5. Install mongo shell:
+```sh
+sudo apt-get install -y mongodb-org
+```
+6. Download certificate:
+```sh
+wget https://truststore.pki.rds.amazonaws.com/global/global-bundle.pem
+```
+7. Connect to mongo:
```

Comment:
to the database

Reply:
@mishushakov please check now. Updated them as per your suggestions.